examples/Currency Conversion display.ipynb

```diff
@@ -0,0 +1,133 @@
+      "File \u001b[0;32m~/.local/lib/python3.8/site-packages/requests/adapters.py:519\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[0;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[1;32m    515\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(e\u001b[38;5;241m.\u001b[39mreason, _SSLError):\n\u001b[1;32m    516\u001b[0m         \u001b[38;5;66;03m# This branch is for urllib3 v1.22 and later.\u001b[39;00m\n\u001b[1;32m    517\u001b[0m         \u001b[38;5;28;01mraise\u001b[39;00m SSLError(e, request\u001b[38;5;241m=\u001b[39mrequest)\n\u001b[0;32m--> 519\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mConnectionError\u001b[39;00m(e, request\u001b[38;5;241m=\u001b[39mrequest)\n\u001b[1;32m    521\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m ClosedPoolError \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m    522\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mConnectionError\u001b[39;00m(e, request\u001b[38;5;241m=\u001b[39mrequest)\n",
+      "\u001b[0;31mConnectionError\u001b[0m: HTTPConnectionPool(host='www.bankofcanada.ca', port=80): Max retries exceeded with url: /valet/observations/FXUSDCAD/csv (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fab9c1d49d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"
+     ]
+    }
+   ],
+   "source": [
+    "data = requests.get(url).text.strip().splitlines()[8:]\n",
+    "keys = data.pop(0)\n",
+    "keys = keys.split(',')\n",
+    "keys = [i.strip('\"').strip(\"'\").strip() for i in keys]\n",
+    "\n",
+    "for row in csv.DictReader(data, keys):\n",
+    "    print(row)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.10"
+  },
+  "parameterise": {
+   "stock": {
+    "display_name": "Stock symbol"
+   }
+  }
```

Comment:
This doesn't make sense, because there's no `stock` parameter - I'm guessing the notebook started as a copy of another one in this folder.

Reply:
yeah sorry this is stuff i should have cleaned up before putting in a pr